dts/bindings/sensor/sensor-device.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sensor Device
+
+include: base.yaml
+
+properties:
+  vendor:
+    type: string
+    required: false
+
+  model:
+    type: string
+    required: false
+
+  friendly-name:
```

Comment:
How is `friendly-name` different from `model`?

Reply:
Friendly name is a standard HID sensor property. If we have two instances of the same model mounted in different locations (e.g., a lid accelerometer and a base accelerometer in a laptop), we can distinguish them from each other when we enumerate sensors to the host OS. We could maybe use `label` for this purpose?

Reply:
> We could maybe use label for this purpose?

Seems like a legitimate usage of `label` to me. @galak?